src/site/content/en/blog/smoothness/index.md

```diff
@@ -0,0 +1,492 @@
+
+### Detecting animations
+
+High frame throughput matters especially during periods with important
+animations. Different animation types will depend on visual updates from a
+specific thread (main, compositor, or a worker), so its visual update is
+dependent on that thread providing its update within the deadline. We say that a
+given thread _affects smoothness_ whenever there is an active animation that
+depends on that thread update.
+
+Some types of animations are easier to define and detect than others.
+Declarative animations, or user-input-driven animations, are clearer to define
+than JavaScript-driven animations implemented as periodic updates to animatable
+style properties.
+
+Even with `requestAnimationFrame()`, which is designed for animations, you
+cannot always assume that every rAF call is necessarily producing a visual
+update or animation. For example, using rAF polling just to track frame rate
+(as shown above) should not itself affect smoothness measurements since there is
+no visual update.
+
+{% Aside %}
+  **Note:** animation detection in tooling, as well as the specifics of
+  animation implementations in browsers, continually evolves and improves.
+  Chrome recently moved [background-color
+  animations](https://developer.chrome.com/blog/hardware-accelerated-animations/#whats-coming-next)
+  from main thread to compositor!
+{% endAside %}
+
+### Quality vs. quantity
+
+Finally, detecting animations and animation frame updates is still only part of
+the story because it only captures the quantity of animation updates, not the
+quality.
+
+For example, you may see a perfectly steady stream of 60fps while watching a
+video. Technically, this is perfectly smooth. But the video itself may have a
+low bit rate, or even issues with network buffering. This isn't captured by
+animation smoothness metrics directly, yet may still be quite jarring to the
+user.
+
+Or, a game which leverages `<canvas>` (perhaps even using techniques like
+[offscreen
+canvas](https://developers.google.com/web/updates/2018/08/offscreen-canvas) to
+ensure a steady frame rate) may technically be perfectly smooth in terms of
+animation frames, all while failing to load high quality game assets into the
+scene or exhibiting rendering artifacts.
+
+And of course, a site can just have some really lame animations :)
+
+<img
+src="https://upload.wikimedia.org/wikipedia/commons/1/1c/Under-Construction-Bulldozer.gif"
+alt="Old school under construction GIF" width="410" height="49">
+
+I mean, I guess they were pretty cool for their time!
+
+## States of a single animation frame
+
+Because frames may be partially presented, or dropped frames may happen in ways
+that do not affect smoothness, we have begun to think of each frame as having a
+completeness or smoothness score.
+
+Here are the ways in which we interpret the state of a single animation frame:
+
+{% Img src="image/eqprBhZUGfb8WYnumQ9ljAxRrA72/E0O8AhtBfv1XPdgTSNxX.svg",
```

Comment:
Accessibility alert, text as an image without an appropriate alt. This should really just go into plain text.

Reply:
It was originally SVG because I wasn't sure if we could do the coloring, but I think we probably can. Let me see if that works better...

Reply:
I'm converting it to a table and will ping when done. Then let me know if it's still unclear.

Reply:
Updated! Let me know what you think.

Reply:
Way better! LGTM 👍